internal/pkg/externalplugins/labelblocker/labelblocker.go

```diff
@@ -126,24 +148,45 @@ func handle(cfg *externalplugins.Configuration, ctx labelCtx, gc githubClient, l
 			continue
 		}
 
+		var operate string
+
 		// Undo the illegal operation.
 		if ctx.action == LabeledAction {
+			// Remove the label added illegally.
 			err := gc.RemoveLabel(owner, repo, ctx.number, ctx.label)
 
 			if err == nil {
-				log.Infof("Remove %s label added illegally", ctx.label)
+				operate = fmt.Sprintf("Remove %s label added illegally", ctx.label)
+				log.Infof(operate)
 			} else {
 				return fmt.Errorf("failed to remove illegal label added illegally, %s", err)
 			}
 		} else if ctx.action == UnlabeledAction {
+			// Restore the label removed illegally.
 			err := gc.AddLabel(owner, repo, ctx.number, ctx.label)
 
 			if err == nil {
-				log.Infof("Restore %s label removed illegally", ctx.label)
+				operate = fmt.Sprintf("Restore %s label removed illegally", ctx.label)
+				log.Infof(operate)
 			} else {
 				return fmt.Errorf("failed to restore the illegally removed label, %s", err)
 			}
 		}
+
+		// Leave a reply to explain why do this.
+		msg := fmt.Sprintf("%s, only trusted users and members of trusted teams can do it.\n"+
```

Comment:
I think this message should be configurable.

Reply:
But how to judge whether it is labeled or unlabeled?

Reply:
Depends on the message to describe.

Reply:
> I think this message should be configurable.

Do you mean that it can configure the content of the response like autoresponder?

And the trusted user and trusted team are fixed in the detail part of the template, which is not configurable.

Reply:
We can make it a golang template, by how the user feels about using these configuration items. What do you think? I think it's better to write in details what events are responded to?